content/concurrency.md

```diff
@@ -1,14 +1,57 @@
+##### Examples
 
-This does not exclude individual processes from handling their own internal multiplexing, via threads inside the runtime VM, or the async/evented model found in tools such as [EventMachine](https://github.com/eventmachine/eventmachine), [Twisted](http://twistedmatrix.com/trac/), or [Node.js](http://nodejs.org/).  But an individual VM can only grow so large (vertical scale), so the application must also be able to span multiple processes running on multiple physical machines.
+Web apps have taken a variety of process-execution forms. For example,
+PHP processes run as child processes of Apache, started on demand as needed
+by request volume. Java processes take the opposite approach, with the JVM
+providing one massive uberprocess that reserves a large block of system
+resources (CPU and memory) on startup, with concurrency managed internally
+via threads.
 
-The process model truly shines when it comes time to scale out.  The [share-nothing, horizontally partitionable nature of twelve-factor app processes](./processes) means that adding more concurrency is a simple and reliable operation.  The array of process types and number of processes of each type is known as the *process formation*.
+This does not exclude individual processes from handling their own internal
+multiplexing, via threads inside the runtime VM, or the async/evented model
+found in tools such as [EventMachine](https://github.com/eventmachine/eventmachine),
+[Twisted](http://twistedmatrix.com/trac/), or
+[Node.js](http://nodejs.org/).
 
-Twelve-factor app processes [should never daemonize](http://dustin.github.com/2010/02/28/running-processes.html) or write PID files.  Instead, rely on the operating system's process manager (such as [systemd](https://www.freedesktop.org/wiki/Software/systemd/), a distributed process manager on a cloud platform, or a tool like [Foreman](http://blog.daviddollar.org/2011/05/06/introducing-foreman.html) in development) to manage [output streams](./logs), respond to crashed processes, and handle user-initiated restarts and shutdowns.
+##### Guidance
+
+An individual VM can only grow so large (vertical scale), so the application
+must also be able to span multiple processes running on multiple physical or
+virtual machines.
+
+#### 2\. A twelve-factor app scales out by assigning workloads to separate process types.
+
+The process model truly shines when it comes time to scale out.
+The [share-nothing, horizontally partitionable nature of twelve-factor app processes](./processes)
+means that adding more concurrency is a simple and reliable operation.
+The array of process types and number of processes of each type is known as
+the *process formation*.
```

Comment:
The word "formation" here is tricky, because the word has two definitions:

* a structure or arrangement of something.
   * a formal arrangement of aircraft in flight or troops.
   * an assemblage of rocks or series of strata having some common characteristic.
* the action of forming or process of being formed.

I'm assuming you mean the first set of definitions, but my mind went to the second.